CorsixTH/Lua/humanoid_actions/die.lua

```diff
@@ -261,8 +261,6 @@ local function action_die_start(action, humanoid)
   local anims = humanoid.die_anims
   assert(anims, "Error: no death animation for humanoid ".. humanoid.humanoid_class)
   action.must_happen = true
-  -- TODO: Right now the angel version of death is the only possibility
-  -- The Grim Reaper should sometimes also have a go.
   local fall = anims.fall_east
 
   --If this isn't done their bald head will become bloated instead of suddenly having hair:
```

Comment:
This animation is recorded in #495 / #1638 but there appears to be code now.

Reply:
The todo is from 11 years ago and is resolved, the Reaper is implemented. Can't speak to those issues.

Reply:
Sorry I meant the comment at line 266.